lms/djangoapps/certificates/views.py

```diff
@@ -245,6 +245,16 @@ def render_html_view(request):
     context = {}
     configuration = CertificateHtmlViewConfiguration.get_config()
     context = configuration.get('default', {})
+    active_certificate = None
+
+    def _get_certificate_signatories(certificate):
+        """
+        Return a list of signatories of a certificate.
+        """
+        signatories = []
+        if certificate.get('signatories', False):
```

Comment:
Optional: Wouldn't `signatories = certificate.get('signatories', [])` accomplish the same result - indeed I think that this means we don't really need this method herein, and can simply call
`context['signatories'] = active_certificate.get('signatories', [])`
below (line 471)